manage_breast_screening/notifications/tests/integration/test_create_appointments_from_azure_storage.py

```diff
@@ -98,3 +101,7 @@ def test_files_are_processed_in_correct_order(self, helpers):
         assert booked_then_cancelled.cancelled_at is not None
         assert assessment.created_at < booked_then_cancelled.updated_at
         assert booked_then_cancelled.booked_at < booked_then_cancelled.cancelled_at
+        assert booked_then_cancelled.originating_file == [
```

Comment:
Perhaps we can add a test which demonstrates how we can retrieve the appointments a file has created or updated?
It's good to be able to know which files created or updated the given record but can we find all the records created or updated by a file? I would imagine this to be important for a support scenario.